dali/operators/reader/nvdecoder/sequencewrapper.h

```diff
@@ -97,26 +76,18 @@ struct SequenceWrapper {
 
   Tensor<GPUBackend> sequence;
 
-  int count;
-  int height;
-  int width;
-  int channels;
-  int label;
+  int count = -1;
+  int height = -1;
+  int width = -1;
+  int channels = -1;
+  int label = -1;
   vector<double> timestamps;
-  int first_frame_idx;
-  DALIDataType dtype;
+  int first_frame_idx = -1;
+  DALIDataType dtype = {};
```

Comment:
```suggestion
  DALIDataType dtype = DALI_NO_TYPE;
```
The default is coincidentally DALI_UINT8 - do we want that?

Reply:
I don't think we care that much, but changed.